docs/testrunner-toolkit/configuration/espresso.md

```diff
@@ -231,6 +231,9 @@ testOptions:
   size: small
   package: com.example.android.testing.androidjunitrunnersample
   annotation: com.android.buzz.MyAnnotation
+  numShards: 4
+  shardIndex: 1
```

Comment:
I am not super clear on how these settings work together....if you are "creating" shards, then how do you know which index to set? Maybe we can have a quick chat to clarify?

Reply:
yeah. sure!